Documentation/k8s-filesystem.md

```diff
@@ -0,0 +1,102 @@
+# Shared File System Quickstart
+
+A shared file system can be mounted read-write from multiple pods. This may be useful for applications which can be clustered using a shared filesystem. 
+
+This example runs a shared file system for the [kube-registry](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/registry).
+
+### Prerequisites
+
+This guide assumes you have created a Rook cluster and pool as explained in the main [Kubernetes guide](kubernetes.md)
+
+## Rook Client
```

Comment:
i was erring on the side of having the steps in one place, but i does seem like too much duplication. i'll look again at just a link.